src/app.html

```diff
@@ -1,6 +1,4 @@
 <template>
     <require from="./css/styles.css"></require>
-    <require from="./commons/navbar-searchbar/navbar-searchbar"></require>
-    <navbar-searchbar></navbar-searchbar>
```

Comment:
this navbar is meant to be a common element, where the user can change apps and workspaces.. Can you please revert this deletion.

Reply:
Thanks for this comment, I have totally forgotten about that deletion. Done